src/main/java/com/datadoghq/datadog_lambda_java/EnhancedMetric.java

```diff
@@ -36,6 +36,7 @@
             m.put("account_id", accountId);
             m.put("memorysize", ctx.getMemoryLimitInMB());
             m.put("cold_start", ColdStart.getColdStart(ctx));
+            m.put("datadog_lambda_version", BuildConfig.datadog_lambda_version);
```

Comment:
Should this be `datadog_lambda` to stay consistent with https://github.com/DataDog/datadog-lambda-python/pull/87/files#diff-9ece8c6cdb00f36112624da37214e888R252 ?

Reply:
fixed!